_includes/ee-linux-install-reuse.md

```diff
@@ -340,4 +340,4 @@ You must delete any edited configuration files manually.
 
 - Continue with the [User Guide](/engine/userguide/index.md).
 
-{% endif %}
\ No newline at end of file
+{% endif %}
```

Comment:
`\ No newline at end of file`

couldn't figure out how to get rid of this in three minutes of googling.  feel free to edit/carry as needed.

Reply:
Yey stackoverflow https://stackoverflow.com/a/16114535.  Re-pushed without whitespace shenanigans.